lib/cartopy/mpl/geoaxes.py

```diff
@@ -44,7 +45,7 @@
 from cartopy.mpl.slippy_image_artist import SlippyImageArtist
 
 
-assert mpl.__version__ >= '3.4', \
+assert packaging.version.parse(mpl.__version__).release[:2] >= (3, 4), \
```

Comment:
I guess mpl v3.10 will ship next year, at which point the string comparison would become problematic.